apps/framework-cli/src/framework/consumption-helpers.ts

```diff
@@ -0,0 +1,141 @@
+const typeMapping = {
+  Int: "number",
+  UInt: "number",
+  UInt8: "number",
+  LowCardinality: "string",
+  UInt16: "number",
+  UInt32: "number",
+  UInt64: "string",
+  UInt128: "string",
+  UInt256: "string",
+  Int8: "number",
+  Int16: "number",
+  Int32: "number",
+  Int64: "string",
+  Int128: "string",
+  Int256: "string",
+  Float32: "number",
+  Float64: "number",
+  Decimal: "number",
+  Bool: "boolean",
+  String: "string",
+  FixedString: "string",
+  UUID: "string",
+  Date32: "string",
+  Date64: "string",
+  DateTime32: "string",
+  DateTime64: "string",
+  IPv4: "string",
+  IPv6: "string",
+};
+
+export const mapFromClickHouseType = (clickHouseType: string) => {
+  return typeMapping?.[clickHouseType] || "string";
+};
+
+/**
+ * Convert the JS type (source is JSON format by API query parameter) to the corresponding ClickHouse type for generating named placeholder of parameterized query.
+ * Only support to convert number to Int or Float, boolean to Bool, string to String, other types will convert to String.
+ * If exist complex type e.g: object, Array, null, undefined, Date, Record.. etc, just convert to string type by ClickHouse function in SQL.
+ * ClickHouse support converting string to other types function.
+ * Please see Each section of the https://clickhouse.com/docs/en/sql-reference/functions and https://clickhouse.com/docs/en/sql-reference/functions/type-conversion-functions
+ * @param value
+ * @returns 'FLoat', 'Int', 'Bool', 'String'
+ */
+
+export const mapToClickHouseType = (value: any) => {
+  if (typeof value === "number") {
```

Comment:
Maybe constants or an enum could be useful for these strings and in `typeMapping` just to avoid any possible typos in the future.